src/pykeen/triples/triples_factory.py

```diff
@@ -382,11 +388,16 @@ def triples(self) -> np.ndarray:  # noqa: D401
 
     def extra_repr(self) -> str:
         """Extra representation string."""
-        return (
-            f"num_entities={self.num_entities}, "
-            f"num_relations={self.num_relations}, "
-            f"num_triples={self.num_triples}, "
-            f"inverse_triples={self.create_inverse_triples}"
+        d = [
+            ('num_entities', self.num_entities,),
+            ('num_relations', self.num_relations),
+            ('num_triples', self.num_triples),
+            ('inverse_triples', self.create_inverse_triples),
+        ]
+        d.extend(sorted(self.metadata.items()))
```

Comment:
Only sort by key.

By default, tuples are sorted by all entries. If any of the values are not comparable, this will raise an error.

Reply:
True, but keys are unique so it will never have a sort collision on the values. This way I can do it in a one-liner :)